Mage/src/main/java/mage/constants/SpellAbilityCastMode.java

```diff
@@ -13,7 +13,8 @@ public enum SpellAbilityCastMode {
     MADNESS("Madness"),
     FLASHBACK("Flashback"),
     BESTOW("Bestow"),
-    DISTURB("Disturb");
+    DISTURB("Disturb"),
+    MORE_THAN_MEETS_THE_EYE("More Than Meets The Eye");
```

Comment:
No card hacks allows in such game engine's deep level. Must be another way to implement your ability.

Reply:
It's not a hack, it's correcting More Than Meets The Eye (already implemented that deep-game-engine level way by @theelk801 ) to be actually (correctly) distinct from Disturb which it was based on.

Reply:
As @Grath mentions, I was simply trying to separate from disturb copy paste remnants, I can revert that part if need be.

Reply:
OK, I see, you can keep it. But must fix:
1. Add new type to `case DISTURB` code (search it);
2. Use `setCardName` with additional name like `+ " with More Than Meets the Eye"`